packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -353,8 +373,8 @@ export default class SalesforceCRMService implements CRM {
     }
   }
 
-  public async deleteEvent(uid: string) {
-    const deletedEvent = await this.salesforceDeleteEvent(uid);
+  public async deleteEvent(uid: string, event: CalendarEvent) {
+    const deletedEvent = await this.salesforceDeleteEvent(uid, event);
```

Comment:
We need to pass the event object to the delete method to get the data needed if the user wants to write booking data to the event object on delete.